src/main/webapp/script.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+getOptions();
+
+let selected = [];
+let courses = [];
+
+/**
+ * Gets courses from /courselist servlet to populate dropdown list
+ */
+async function getOptions() {
+  const response = await fetch('/courselist');
+  courses = await response.json();
+  const courseContainer = document.getElementById('courses');
+  courseContainer.innerHTML = '';
+  createOption('Select a Course', 'courses', /*setValue=*/ false);
+  courses.forEach(course => createOption(course, 'courses', /*setValue=*/ true));
+}
+
+/**
+ * Creates options in select list
+ * @param {string} course The name of the course to add to the dropdown
+ * @param {string} container The id name of the container you want to add options to
+ * @param {boolean} setValue Whether to set the value of the option to the string 
+ */
+function createOption(course, container, setValue) {
```

Comment:
maybe `addOption`? `createOption` implies that you would return the option, and not modify any of the arguments.